org_runners.tf

```diff
@@ -0,0 +1,27 @@
+resource "kubectl_manifest" "github_org_runners" {
+  for_each  = { for org in var.github_org_runners : org.name => org }
+  yaml_body = <<YAML
```

Comment:
Might have an easier time in the future to manipulate values using `yamlencode`.

Reply:
it's better to convert to this right ?

<img width="670" alt="image" src="https://user-images.githubusercontent.com/46478191/181675997-cd88965b-5855-4021-b0ae-01b505ea202d.png">

Reply:
Yes.

Reply:
done.